### PR TITLE
feat: list our popular open source projects

### PR DESCRIPTION
**Pull request by dunglas**
People go on GitHub mostly to find code.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -3,7 +3,19 @@
 <a href="https://les-tilleuls.coop"><img align="right" src="https://raw.githubusercontent.com/coopTilleuls/.github/main/profile/public/img/tree.svg" height="250" alt="Les-Tilleuls.coop"></a>
 
 Les-Tilleuls.coop is a tech [worker cooperative](https://en.wikipedia.org/wiki/Worker_cooperative) specialized in e-commerce, trainings and free software. We bring your projects to life by combining creativity, durability and cutting-edge development.
-In this page, you will find all our internal documentation and useful information.
+
+If you're interested in the free software we create, check out their dedicated GitHub pages:
+
+* [🕷 API Platform](https://github.com/api-platform): create REST and GraphQL APIs, scaffold Jamstack webapps, stream changes in real-time, written in PHP and TypeScript
+* [🎼 Our Symfony components](https://github.com/symfony): a set of reusable PHP components and a framework for web projects
```
Comment (soyuka) [inline profile/README.md:10]:
```suggestion
* [🎼 Symfony components we contribute to](https://github.com/symfony): a set of reusable PHP components and a framework for web projects
```

File: profile/README.md [hunk 2]
```diff
@@ -3,7 +3,19 @@
...
+* [💌 Mercure](https://github.com/dunglas/mercure): real-time made easy, open protocol and reference implementation written in Go
+* [🔨 Vulcain](https://github.com/dunglas/vulcain): fast and idiomatic client-driven REST APIs, open protocol and reference implementation written in Go
+* [⚡️ React ESI](https://github.com/dunglas/react-esi): blazing-fast Server-Side Rendering for React and Next.js, written in TypeScript
+* [📄 Doctrine JSON ODM](https://github.com/dunglas/doctrine-json-odm): An Object-Document Mapper (ODM) for Doctrine ORM leveraging new JSON types of modern RDBMS
+* [🏄 ESQL](https://github.com/soyuka/esql): PHP Extended SQL
+* [📦 Our other projects](https://github.com/orgs/coopTilleuls/repositories)
+
```
Comment (soyuka) [inline profile/README.md:17]: WDYT in adding something like this:

We have many open source advocates, here are some profiles you may want to follow:

* [dunglas](https://github.com/dunglas)
* [chalasr](https://github.com/chalasr) 
* [soyuka](https://github.com/soyuka)
* [Alan Poulain](https://github.com/alanpoulain)
* [Vincent Chalamon](https://github.com/vincentchalamon)

File: profile/README.md [hunk 3]
```diff
@@ -3,7 +3,19 @@
...
-In this page, you will find all our internal documentation and useful information.
+
+If you're interested in the free software we create, check out their dedicated GitHub pages:
+
+* [🕷 API Platform](https://github.com/api-platform): create REST and GraphQL APIs, scaffold Jamstack webapps, stream changes in real-time, written in PHP and TypeScript
+* [🎼 Our Symfony components](https://github.com/symfony): a set of reusable PHP components and a framework for web projects
+* [💌 Mercure](https://github.com/dunglas/mercure): real-time made easy, open protocol and reference implementation written in Go
+* [🔨 Vulcain](https://github.com/dunglas/vulcain): fast and idiomatic client-driven REST APIs, open protocol and reference implementation written in Go
+* [⚡️ React ESI](https://github.com/dunglas/react-esi): blazing-fast Server-Side Rendering for React and Next.js, written in TypeScript
+* [📄 Doctrine JSON ODM](https://github.com/dunglas/doctrine-json-odm): An Object-Document Mapper (ODM) for Doctrine ORM leveraging new JSON types of modern RDBMS
```
Comment (soyuka) [inline profile/README.md:14]:
```suggestion
* [📄 Doctrine JSON ODM](https://github.com/dunglas/doctrine-json-odm): An Object-Document Mapper (ODM) for Doctrine ORM leveraging new JSON types of modern RDBMS
* [🛡️ Oauth2 server bundle](https://github.com/chalasr/oauth2-server-bundle): OAuth2 implementation 
```

File: profile/README.md (hunk 2 above)
Comment (dunglas) [reply profile/README.md:17]: We could maybe just add a like to the "people" page?

File: profile/README.md [hunk 4]
```diff
@@ -3,7 +3,19 @@
 <a href="https://les-tilleuls.coop"><img align="right" src="https://raw.githubusercontent.com/coopTilleuls/.github/main/profile/public/img/tree.svg" height="250" alt="Les-Tilleuls.coop"></a>
 
 Les-Tilleuls.coop is a tech [worker cooperative](https://en.wikipedia.org/wiki/Worker_cooperative) specialized in e-commerce, trainings and free software. We bring your projects to life by combining creativity, durability and cutting-edge development.
-In this page, you will find all our internal documentation and useful information.
+
+If you're interested in the free software we create, check out their dedicated GitHub pages:
```
Comment (dunglas) [inline profile/README.md:7]:
```suggestion
If you're interested in the free software we back, check out their dedicated GitHub pages:
```

Alternative proposal to @soyuka's changes.

File: profile/README.md (hunk 1 above)
Comment (dunglas) [inline profile/README.md:10]:
```suggestion
* [🎼 Symfony](https://github.com/symfony): a set of reusable PHP components and a framework for web projects
```

File: profile/README.md [hunk 5]
```diff
@@ -3,7 +3,21 @@
...
-In this page, you will find all our internal documentation and useful information.
+
+If you're interested in the free software we back, check out their dedicated GitHub pages:
+
+* [🕷 API Platform](https://github.com/api-platform): create REST and GraphQL APIs, scaffold Jamstack webapps, stream changes in real-time, written in PHP and TypeScript
+* [🎼 Symfony](https://github.com/symfony): a set of reusable PHP components and a framework for web projects
+* [💌 Mercure](https://github.com/dunglas/mercure): real-time made easy, open protocol and reference implementation written in Go
+* [🔨 Vulcain](https://github.com/dunglas/vulcain): fast and idiomatic client-driven REST APIs, open protocol and reference implementation written in Go
+* [⚡️ React ESI](https://github.com/dunglas/react-esi): blazing-fast Server-Side Rendering for React and Next.js, written in TypeScript
+* [💅 PHP Standard Library](https://github.com/azjezz/psl): a modern, consistent, centralized, well-typed, non-blocking set of APIs for PHP programmers.
```
Comment (azjezz) [inline profile/README.md:14]:
```suggestion
* [📚 PHP Standard Library](https://github.com/azjezz/psl): a modern, consistent, centralized, well-typed, non-blocking set of APIs for PHP programmers. 
```

fits better 😛

Comment (dunglas) [reply profile/README.md]: Are you sure? 😛

**Review by azjezz — APPROVED** [review]

File: profile/README.md [hunk 6]
```diff
@@ -3,7 +3,21 @@
...
+* [🎼 Symfony](https://github.com/symfony): a set of reusable PHP components and a framework for web projects
+* [💌 Mercure](https://github.com/dunglas/mercure): real-time made easy, open protocol and reference implementation written in Go
+* [🔨 Vulcain](https://github.com/dunglas/vulcain): fast and idiomatic client-driven REST APIs, open protocol and reference implementation written in Go
+* [⚡️ React ESI](https://github.com/dunglas/react-esi): blazing-fast Server-Side Rendering for React and Next.js, written in TypeScript
+* [📚 PHP Standard Library](https://github.com/azjezz/psl): a modern, consistent, centralized, well-typed, non-blocking set of APIs for PHP programmers. 
+* [📄 Doctrine JSON ODM](https://github.com/dunglas/doctrine-json-odm): an Object-Document Mapper (ODM) for Doctrine ORM leveraging new JSON types of modern RDBMS
+* [🛡️ OAuth2 Server Bundle](https://github.com/thephpleague/oauth2-server-bundle): a spec compliant, secure by default PHP OAuth 2.0 Server for Symfony 
+* [🏄 ESQL](https://github.com/soyuka/esql): PHP Extended SQL
+* [📦 Our other projects](https://github.com/orgs/coopTilleuls/repositories)
```
Comment (alanpoulain) [inline profile/README.md:18]: Shouldn't we highlight some?
- https://github.com/coopTilleuls/CoopTilleulsForgotPasswordBundle
- https://github.com/coopTilleuls/UrlSignerBundle
- https://github.com/coopTilleuls/CoopTilleulsSyliusClickNCollectPlugin

File: profile/README.md [hunk 7]
```diff
@@ -3,7 +3,21 @@
...
-In this page, you will find all our internal documentation and useful information.
+
+If you're interested in the free software we back, check out their dedicated GitHub pages:
+
+* [🕷 API Platform](https://github.com/api-platform): create REST and GraphQL APIs, scaffold Jamstack webapps, stream changes in real-time, written in PHP and TypeScript
+* [🎼 Symfony](https://github.com/symfony): a set of reusable PHP components and a framework for web projects
+* [💌 Mercure](https://github.com/dunglas/mercure): real-time made easy, open protocol and reference implementation written in Go
+* [🔨 Vulcain](https://github.com/dunglas/vulcain): fast and idiomatic client-driven REST APIs, open protocol and reference implementation written in Go
+* [⚡️ React ESI](https://github.com/dunglas/react-esi): blazing-fast Server-Side Rendering for React and Next.js, written in TypeScript
+* [📚 PHP Standard Library](https://github.com/azjezz/psl): a modern, consistent, centralized, well-typed, non-blocking set of APIs for PHP programmers.
```
Comment (alanpoulain) [inline profile/README.md:14]:
```suggestion
* [📚 PHP Standard Library](https://github.com/azjezz/psl): a modern, consistent, centralized, well-typed, non-blocking set of APIs for PHP programmers
```

**Pull request merged**